### ISSUE TITLE
Capstone proposal

### ISSUE THREAD

**Issue by mzakariaCERN** [paigewil/capstone#2]
In the first paragraph, I would rephrase the introduction to be a bit less charged. This is intended to show a skill set, not make a statement 👍 

@paigewil

**Comment by paigewil** [comment]
@mzakariaCERN 
I updated my intro, but am not sure if it's less charged or not, so before I close this issue, I wanted to have you check it and make sure it was good.

Thanks!

**Comment by mzakariaCERN** [comment]
Seems better now. We will revisit as we progress with the project 👍
